spec/subnets-lifecycle.md

```diff
@@ -0,0 +1,68 @@
+# Subnet Lifecycle
+A subnet is a new subsystem that a user can spawn from a parent subnet in a permissionless and on-demand way, depending on scalability requirements. Subnets have separate consensus algorithms and cryptoeconomic rules from their parent subnet. Subnets are firewalled from the parent network.
```

Comment:
> A subnet is a new subsystem that a user can spawn from a parent subnet

Recursive definition. I guess this should mention that subnet is a chain of its own, and one can think of a subnet as an `LN+1` connected to a `LN` chain, eg `L2` is connected to `L1`.

> Subnets are firewalled from the parent network.

Is "parent network" different from "parent subnet"?

Reply:
they are the same

Reply:
Then I think it's best to name them the same to avoid confusion. As this is the spec, it should be very dry and precise, IMO.